src/CgiHandler.cpp

```diff
@@ -38,14 +35,52 @@ static char **transferToString(std::map<std::string, std::string> cgiEnvironment
 	return environmentString;
 }
 
-int CgiHandler::executeCgi(HttpRequest request)
+std::string findShebang(std::string fullPath)
 {
+	std::ifstream file(fullPath);
+	std::string	line;
 	
-	std::map<std::string, std::string> cgiEnvironment = initCgiEnvironment(request);
-	char **variableArray = transferToString(cgiEnvironment);
+	if (file.is_open() == true)
```

Comment:
we should follow the filehandlers way of doing this like if the file is not open or is bad etc we throw right?

Reply:
sure, I'll fix it for the next one!